{
  MockEndpoint error=context.getEndpoint("mock:error",MockEndpoint.class);
  error.expectedMessageCount(1);
  sendTimeOutMessage("direct:doCatch");
  error.assertIsSatisfied();
}
